### PR TITLE
v23 run generate analysis files (1/n)

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
Prep for v23 release

#### What was your approach?
Run `generate-analysis-files.sh` for V23

#### What GitHub issue does your pull request address?
NA

#### Is there anything that you want to discuss further?
Will look into the changes in files to see if something is strikingly wrong or maybe a change was missed previously.
Update: these look ok, mainly due  to updates in independent specimens and notebook HTML files. The `cnv_consensus.tsv` file has changed, but the final CNV files used in release do not change: 
```
fa0adcd26f408d840f25339d2a1ea09f  consensus_seg_annotated_cn_autosomes.tsv.gz
9589cd18d0e6c1f7c2d939126c63ec6c  consensus_seg_annotated_cn_x_and_y.tsv.gz
b8b97483b4d65e65c8ae34ff89b3ef94  consensus_seg_with_status.tsv
b9284650be04df3538e6c6dba29b8eb0  pbta-cnv-consensus.seg.gz
```

I am going to investigate the SNV consensus maf changes, but my hunch is the row ordering is the cuplrit because I would also not expect those to change.

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
yes

#### What is your summary of the results?
Files being updated in release:
```
fusion_summary_ependymoma_foi.tsv (expected with #1619)
ae559544f5e8baf0f0f21ab7c00f0041  independent-specimens.rnaseq.primary-plus-stranded.tsv
44b1e1d483465798221799cf63b93fb8  independent-specimens.wgs.primary-plus.tsv
43bf4bd1f0d073607276ce6eef989951  independent-specimens.wgs.primary.tsv
372db726c453efded2340da8ad536e81  independent-specimens.wgswxs.primary-plus.tsv
94283581188cc87427b3b58b1fe75860  independent-specimens.wgswxs.primary.tsv
d4251fcd7f7bea0f64a9a247a40a21e0  pbta-cnv-consensus-gistic.zip
757159a9d864d78ef65c8b68453e2f86  pbta-fusion-recurrently-fused-genes-byhistology.tsv
95d6b0c3401f8c6c10c4c013cb78e275  pbta-fusion-recurrently-fused-genes-bysample.tsv
2be7929f8fc130fc2048cf8f8d0b1c55  pbta-snv-consensus-mutation.maf.tsv.gz
21126513a05c43427af774884aaeeb46  tcga-snv-consensus-snv.maf.tsv.gz
```

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
@jharenza the CI file-related updates should go into a different PR (`analyses/create-subset-files/create_subset_files.sh` and `analyses/create-subset-files/biospecimen_ids_for_subset.RDS`)

File: analyses/fusion_filtering/results/pbta-fusion-recurrent-fusion-bysample.tsv [hunk 1]
```diff
@@ -318,7 +319,7 @@ BS_B7GEH89C	0	0	0	1	0
 BS_B7QS4DHK	0	0	0	1	0
 BS_B7X8VXQ3	0	0	1	0	0
 BS_B8T7M0WV	0	0	0	1	0
-BS_B9V8RGTA	0	0	0	1	0
+BS_B9V8RGTA	0	1	0	0	0
```
Comment (jaclyn-taroni) [inline analyses/fusion_filtering/results/pbta-fusion-recurrent-fusion-bysample.tsv:322]: It's not obvious to me that we would expect this change?

**Review by jaclyn-taroni — COMMENTED** [review]
In general, I will wait to hear back from you about the MAF changes. I expect many of the changes are related to the independent specimen files, including those to the fusion results because of:

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/aa54a954cb2b0c5c5342c0c7284275e35a8f5505/analyses/fusion_filtering/06-recurrent-fusions-per-histology.R#L59-L61

However, the new _EWSR1--FLI1_ fusion in a specimen that was already captured in the recurrent fusions file surprised me?

File: analyses/fusion_filtering/results/pbta-fusion-recurrent-fusion-bysample.tsv (hunk 1 above)
Comment (jharenza) [reply analyses/fusion_filtering/results/pbta-fusion-recurrent-fusion-bysample.tsv:322]: This seems to not have been correct in v22, but not sure why. In both v22 and v23 putative oncogenic, we see: 

```r
# A tibble: 4 × 5
  Sample      FusionName  LeftBreakpoint RightBreakpoint Fusion_Type
  <chr>       <chr>       <chr>          <chr>           <chr>      
1 BS_B9V8RGTA EWSR1--FLI1 22:29287134    11:128805366    in-frame   
2 BS_B9V8RGTA EWSR1--FLI1 22:29287134    11:128805366    in-frame   
3 BS_B9V8RGTA EWSR1--FLI1 22:29287134    11:128807180    in-frame   
4 BS_B9V8RGTA EWSR1--FLI1 22:29288786    11:128805366    frameshift 
```

**Comment by jharenza** [comment]
unzipped mafs are identical:

```
harenzaj@38f9d38f36c9 data % md5sum release-v22-20220505/*snv-consensus*maf.tsv   
337cc86a1c62eb2cef3cc9d8669c2eda  release-v22-20220505/pbta-snv-consensus-mutation.maf.tsv
f979aad447c9f8bcbac70b1fd2270a73  release-v22-20220505/tcga-snv-consensus-snv.maf.tsv
harenzaj@38f9d38f36c9 data % md5sum release-v23-20230115/*snv-consensus*maf.tsv
337cc86a1c62eb2cef3cc9d8669c2eda  release-v23-20230115/pbta-snv-consensus-mutation.maf.tsv
f979aad447c9f8bcbac70b1fd2270a73  release-v23-20230115/tcga-snv-consensus-snv.maf.tsv
```

**Review by jaclyn-taroni — APPROVED** [review]
The changes seem to be expected. The zip file change (i.e., the GISTIC results) makes sense since other file attributes are probably being included, regardless of whether or not the underlying results change.

I would not merge this yet, I would wait until all the PRs that will be stacked on this branch are merged into this branch first.

**Pull request merged**